general-terms-of-use.md

```diff
@@ -0,0 +1,97 @@
+# Clever General Terms of Use
+
+_Effective Date: March 19, 2015_
+
+Welcome to Clever! We’re delighted to have you on our website and maybe even using our service (the “Services”). Please continue reading to learn about the terms by which you may use our Services. If you have questions about this Agreement, please contact us at [support@clever.com](mailto:support@clever.com).
```

Comment:
IMO we should split these long lines so that changes can more easily be detected.